imap_processing/ultra/l1b/ultra_l1b_extended.py

```diff
@@ -751,31 +750,34 @@ def determine_species(tof: np.ndarray, path_length: np.ndarray, type: str) -> ND
     return species_bin
 
 
-def get_de_az_el(v: NDArray) -> tuple[NDArray, NDArray]:
+def get_theta_phi(
+    front_position: tuple, back_position: tuple, d: np.ndarray
+) -> tuple[NDArray, NDArray]:
     """
-    Compute azimuth (phi) angles and elevation (theta).
+    Compute phi angles and theta.
```

Comment:
I like the extra detail you had before. Is the theta here a polar angle or elevation angle? Is the range from -90 -> 90 or 0 -> 180?
```suggestion
    Compute the instrument angles phi and theta (azimuth, polar angle).
```